integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -1007,27 +1046,32 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
             &contract_id,
             near_test_contracts::rs_contract().into(),
             &signer,
-            get_shared_block_hash(&node_datas, &test_loop),
+            get_shared_block_hash(&env.datas, &env.test_loop),
         );
-        run_tx(&mut test_loop, deploy_contract_tx, &node_datas, Duration::seconds(5));
+        run_tx(&mut env.test_loop, deploy_contract_tx, &env.datas, Duration::seconds(5));
     }
 
+    let rpc_id = params.rpc_clients[0].clone();
+    let temporary_account =
```

Comment:
1) please add a comment what's this for
2) Do you want this to happen in all tests or should it be part of the TestParams?

Reply:
Added a comment. I would prefer it to happen in all tests, because it currently fails in a test that I did not expect to, so I think it can catch nontrivial bugs.